packages/create-astro/src/index.ts

```diff
@@ -246,7 +312,7 @@ export async function main() {
 		)} to your project using ${bold(cyan('astro add'))}`
 	);
 	await logAndWait('');
-	await logAndWait(`Stuck? Come join us at ${bold(cyan('https://astro.build/chat'))}`, 1000);
+	await logAndWait(`Stuck? Come join us at ${bold(cyan('https://astro.build/chat'))}`, 750);
```

Comment:
Fred talked about a sleep being too long in `create-astro`, however he didn't specify which one. I can only assume it's this one, which did feel long